examples/logical_switch_groups.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+
+from pprint import pprint
+
+from config_loader import try_load_from_file
+from hpOneView.exceptions import HPOneViewException
+from hpOneView.oneview_client import OneViewClient
+
+config = {
+    "ip": "172.16.102.59",
+    "credentials": {
+        "userName": "administrator",
+        "password": ""
+    }
+}
+
+options = {
+    "type": "logical-switch-group",
+    "name": "OneView Test Logical Switch Group",
+    "switchMapTemplate": {
+        "switchMapEntryTemplates": [{
+            "logicalLocation": {
+                "locationEntries": [{
+                    "relativeValue": 1,
+                    "type": "StackingMemberId"
+                }]
+            },
+            "permittedSwitchTypeUri": ""
+        }]
+    }
+}
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+# Get switch type to use in creation of logical switch group
+print("Get switch type to use in creation of logical switch group")
+switch_type = oneview_client.switch_types.get_all()[0]
+print("   Found switch type at uri: '{}'".format(switch_type['uri']))
+
+# Create a logical switch group
+print("Create a logical switch group")
+options['switchMapTemplate']['switchMapEntryTemplates'][
```

Comment:
We are not using the 80 column limit, we are using 120 instead. In this particular case I would prefer avoid line break to improve the readability.